ts/util/html-utils.ts

```diff
@@ -157,13 +157,27 @@ export function makeCopyButton(text: string): HTMLButtonElement {
   return button;
 }
 
-export function makeTextDivWithTooltip(text: string, tooltip: string) {
+export function makeTextDivWithTooltip(text: string, tooltip: string, addCopyButton = false) {
   const div = document.createElement("div");
-  div.appendChild(document.createTextNode(text));
+  div.className = "flex flex-row";
```

Comment:
add `space-x-1` and `items-center` to position this better (the same is done in `makeNameDiv` already)